js/webform_civicrm_options.js

```diff
@@ -64,7 +68,8 @@ if (typeof jQuery.fn.prop !== 'function') {
           $('input.civicrm-enabled, input.select-all-civi-options, input.select-all-civi-defaults').prop('checked', false);
         }
         $('input.civicrm-enabled').change();
-      });
+      }); 
+      */
```

Comment:
There is an extra space behind the ; here -> 
<img width="309" alt="image" src="https://user-images.githubusercontent.com/5340555/129463644-5f1e6552-262e-4aa5-b2b6-5a42b7459f89.png">

Reply:
👍